samples/MonoGame/README.md

```diff
@@ -0,0 +1,90 @@
+# Building a MonoGame app with CoreRT
+
+This document will guide you through compiling a .NET Core [MonoGame](http://www.monogame.net) game with CoreRT.
+
+## Install the .NET Core SDK
+CoreRT is an AOT-optimized .NET Core runtime. If you're new to .NET Core make sure to visit the [official starting page](http://dotnet.github.io). It will guide you through installing pre-requisites and building your first app.
+If you're already familiar with .NET Core make sure you've [downloaded and installed the .NET Core 2 SDK](https://www.microsoft.com/net/download/core).
+
+## Create .NET Core MonoGame project
+Open a new shell/command prompt window and run the following commands.
+```bash
+> dotnet new --install MonoGame.Template.CSharp
+> dotnet new mgdesktopgl -o MyGame
+> cd MyGame
+```
+
+This will install .NET Core MonoGame template and create empty game project. .NET Core MonoGame port lives at https://github.com/cra0zy/MonoGame/tree/core currently. Thank you @cra0zy for the great work! 
+
+Verify that the empty game runs. You should see light blue window:
+
+```bash
+> dotnet run
+```
+
+## Add CoreRT to your project
+Using CoreRT to compile your application is done via the ILCompiler NuGet package, which is [published to MyGet with the CoreRT daily builds](https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.DotNet.ILCompiler).
+For the compiler to work, it first needs to be added to your project.
+
+In your shell/command prompt navigate to the root directory of your project and run the command:
+
+```bash
+> dotnet new nuget 
+```
+
+This will add a nuget.config file to your application. Open the file and in the ``<packageSources> `` element under ``<clear/>`` add the following:
+
+```xml
+<add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+```
+
+Once you've added the package source, add a reference to the compiler by running the following command:
+
+```bash
+> dotnet add package Microsoft.DotNet.ILCompiler -v 1.0.0-alpha-* 
+```
+
+## Restore and Publish your app
+
+Once the package has been successfully added it's time to compile and publish your app! If you're using Windows, make sure you're using `x64 Native Tools Command Prompt for VS 2017` instead of the standard Windows command prompt. In the shell/command prompt window, run the following command:
+
+```bash
+> dotnet publish -r <RID> -c <Configuration>
+```
+
+where `<Configuration>` is your project configuration (such as Debug or Release) and `<RID>` is the runtime identifier, which you specified in the csproj file (one of win-x64, linux-x64, osx-x64). For example, if you want to publish a release configuration of your app for a 64-bit version of Windows the command would look like:
```

Comment:
> which you specified in the csproj file

I think this part became out of date while @A-And was iterating on the ASP.NET version of the doc and we missed updating it.

Reply:
@MichalStrehovsky Guilty. The RuntimeIdentifiers are not necessary - we're sticking to `dotnet publish` conventions similar to the [Raspberry Pi Sample](https://github.com/dotnet/core/blob/master/samples/RaspberryPiInstructions.md).